addons/l10n_my/data/account_tax_template_data.xml

```diff
@@ -11,7 +11,7 @@
         <field name="amount">5.0</field>
         <field name="price_include" eval="0"/>
         <field name="tax_group_id" ref="tax_group_sst" />
-        <field name="invoice_repartition_line_ids" eval="[
+        <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
                 (0,0, {
```

Comment:
```suggestion
        <field name="invoice_repartition_line_ids" eval="[(6, 0, {
```
am I missing something obvious??

Reply:
I had just copied the usage of the CLEAR command for the template line ids from other localization modules. Seemed best to keep things uniform. Can make the change if you think it is better.

Reply:
:man_facepalming: forget about it, I'm an idiot: SET is doing CLEAR + LINK, not CLEAR + CREATE